CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
```

```suggestion
cmake_minimum_required(VERSION 3.15)
```
Any particular reason to require cmake 3.5? I don't think so and `mc_rtc` only requires `3.15`.

---

My bad, the point was just avoiding the minimum required warning from `mc_rtc` and I did not know the minimum was only `3.15`